lib/streamlit/elements/multiselect.py

```diff
@@ -250,6 +276,7 @@ def _multiselect(
         multiselect_proto.form_id = current_form_id(self.dg)
         if help is not None:
             multiselect_proto.help = dedent(help)
+        multiselect_proto.max_selections = max_selections or 0
```

Comment:
nit: Would be nice to move this above the if statement to group it with all of the other things that always set a value in the proto